.env

```diff
@@ -0,0 +1,2 @@
+OVERRIDDEN='main-dot-env-flavor'
+SHARED='shared-key'
```

Comment:
We should see if we can put these into the fixtures directory instead of having them on the top level dir of the project

(they're used only for the tests right?)